### PR TITLE
Add site side unified meta / Twitter card meta / OG tags

### LINKED ISSUE #600: Add Open Graph tags across site
The site should have Open Graph tags attached to each of its pages to describe the content, so that this can be picked up when posting links to social media like facebook or twitter.

We can start by ensuring that entries emit their title and description using Open Graph `<meta>` tags.

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #600
> 
    title: 'Mozilla Pulse'
    description: 'Discover & collaborate on projects for a healthy internet.'
    image: logo

- Facebook debugger (or OG tags): https://developers.facebook.com/tools/debug
- Twitter card validator: https://cards-dev.twitter.com/validator

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -87,13 +87,51 @@ if (NODE_ENV !== `development`) {
   });
 }
 
-function renderPage(appHtml, reactHelmet) {
+function renderPage(appHtml, reactHelmet, canonicalUrl) {
+  const meta = {
+    url: canonicalUrl,
+    title: `Mozilla Pulse`,
+    descritption: `Discover & collaborate on projects for a healthy internet.`,
```
Comment (Pomax) [inline server.js:94]: typo
```suggestion
    description: `Discover & collaborate on projects for a healthy internet.`,
```

File: server.js [hunk 2]
```diff
@@ -87,13 +87,51 @@ if (NODE_ENV !== `development`) {
...
+      width: `600`,
+      height: `600`,
+      altText: `Mozilla logo`
+    }
+  };
+
+  const twitterCard = `
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:title" content="${meta.title}">
+    <meta name="twitter:description" content="${meta.descritption}">
```
Comment (Pomax) [inline server.js:108]:
```suggestion
    <meta name="twitter:description" content="${meta.description}">
```

File: server.js [hunk 3]
```diff
@@ -87,13 +87,51 @@ if (NODE_ENV !== `development`) {
...
+    <meta name="twitter:image" content="${meta.image.url}">
+  `;
+
+  const ogTags = `
+    <meta property="og:url" content="${meta.url}" />
+    <meta property="og:title" content="${meta.title}" />
+    <meta property="og:description" content="${meta.descritption}" />
```
Comment (Pomax) [inline server.js:115]:
```suggestion
    <meta property="og:description" content="${meta.description}" />
```

File: server.js [hunk 4]
```diff
@@ -87,13 +87,51 @@ if (NODE_ENV !== `development`) {
...
+    <meta property="og:image:height" content="${meta.image.height}" />
+    <meta property="og:image:alt" content="${meta.image.altText}" />
+  `;
+
   return `
     <!doctype html>
     <html>
       <head>
+        <title>"${meta.title}"</title>
+        <meta name="description" content="${meta.descritption}">
```
Comment (Pomax) [inline server.js:130]:
```suggestion
        <meta name="description" content="${meta.description}">
```

**Comment by mmmavis** [comment]
fixed typos 😂

**Pull request merged**